### DISCUSSION TITLE
/login/Basic requires username "customer"

### DISCUSSION THREAD

**Discussion by vpatron** [vloschiavo/powerwall2#71] [General]
I was only able to get the login/Basic authentication to work by using the username `customer` and the usual password. [If you have not changed the password, the default is the last 5 characters of the serial number (which starts with "TG")].

Here's the full `curl` request:

```
curl --cacert cacert.pem -s -i -X POST -H "Content-Type: application/json" -d '{"username":"customer","password":"<Last5CharsofSerialNumber>","force_sm_off":false}' https://powerwall/api/login/Basic
```

Hope that helps someone out.

**Comment by agouny** [comment]
Thanks! Been trying for over an hour to login.. it works now :)
